app/containers/MessageBox/index.js

```diff
@@ -111,8 +111,9 @@ export default class MessageBox extends Component {
 	componentDidMount() {
 		const { rid } = this.props;
 		const [room] = database.objects('subscriptions').filtered('rid = $0', rid);
-		if (room.draftMessage && room.draftMessage !== '') {
```

Comment:
don't remove this

Reply:
why? room.draftMessage !== '' is redundant here

Reply:
K.. forgot that empty strings are also false in JS

Reply:
no problem, thanks for taking a look at PR